### PR TITLE
[DOCS] Rules page: advanced sorting and bulk editing

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves:
* #1598
* #1599

Documenting both features in the same PR, since they both involve the Rules page.

Previews:
* [Manage detection rules](https://security-docs_1621.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html)
* [Monitor and troubleshoot rule executions](https://security-docs_1621.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html)

 Several other pages had very minor changes to rename "Activate" to "Enable" (and "Activated" to "Enabled"):
   * https://security-docs_1621.docs-preview.app.elstc.co/guide/en/security/master/upgrade-intro.html
   * https://security-docs_1621.docs-preview.app.elstc.co/guide/en/security/master/post-upgrade-req-cti-alerts.html
   * https://security-docs_1621.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.0.0.html

### CODE REVIEW

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/detections/rules-ui-manage.asciidoc [hunk 1]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
 image::images/install-prebuilt-rules.png[]
 
 [float]
-[[manage-rules-ui]]
-=== Modify existing rules
+[[edit-rules-settings]]
+=== Modify existing rules settings
+
+You can edit the settings of an existing rule. You can also select multiple rules, then edit certain settings in bulk (index patterns and tags).
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:76]:
```suggestion
You can edit an existing rule's settings, and can edit index patterns and tags for multiple rules at once.
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 2]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
 . Do one of the following:
-* Click the *All actions* icon (*...*) on the appropriate row, then select the required action.
-* In the *Rule* column, select all the rules you want to modify, then select the
-required action from the *Bulk actions* menu.
-. To activate or deactivate a rule, click the *Activated* toggle button.
+* Edit a single rule: Click the *All actions* icon (*...*) on a rule, then select *Edit rule settings*. The *Edit rule settings* view opens, where you can modify the <<rules-ui-create, rule's settings>>.
+* Edit multiple rules: Select all the rules you want to edit, then select an action from the *Bulk actions* menu:
+** *Index patterns*: Add or delete the index patterns used by all selected rules.
+** *Tags*: Add or delete tags on all selected rules.
+. On the flyout that opens, enter the settings to change on the rules.
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:86]:
```suggestion
. On the flyout that opens, update the settings.
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 3]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
-<<rule-schedule, rule actions>>, and <<detections-ui-exceptions, add exceptions>>.
+[float]
+[[manage-rules-ui]]
+=== Manage rules
+
+You can duplicate, enable, disable, and delete rules:
+
+. Go to *Detect* -> *Rules*.
+. Do one of the following:
+* Click the *All actions* icon (*...*) on a rule, then select the required action.
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:98]:
```suggestion
* Click the *All actions* icon (*...*) on a rule, then select the desired action.
```

Comment (benironside) [reply docs/detections/rules-ui-manage.asciidoc:98]: just putting it out there — to me required sounds like we are requiring them to do it. Probably not confusing in this context but I still like desire 🤷

File: docs/detections/rules-ui-manage.asciidoc [hunk 4]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
+* Select all the rules you want to modify, then select the required action from the *Bulk actions* menu.
+* To enable or disable a single rule, click the *Enabled* toggle on the rule.
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:100]:
```suggestion
* To enable or disable a single rule, click the rule's *Enabled*/*Disabled* toggle.
```

**Review by benironside — COMMENTED** [review]
Looks great! Left a few small suggestions

**Review by spong — APPROVED** [review]
Overall technical copy LGTM! Note, we've got one more spot where some screenshots may need to be updated as `Activate` has not been switched to `Enable` on the Rule Details page just yet (https://github.com/elastic/kibana/issues/125465).

edit: I've created a PR for updating the Rule Details page's `Activate` to be `Enable`: https://github.com/elastic/kibana/pull/126554

File: docs/detections/rules-ui-manage.asciidoc (hunk 3 above)
Comment (joepeeples) [reply docs/detections/rules-ui-manage.asciidoc:98]: @benironside I'm not a fan of "desire" in technical docs, but definitely agree that "required" isn't right. I think we can omit an adjective altogether :
```suggestion
* Click the *All actions* icon (*...*) on a rule, then select an action.
```

File: docs/detections/rules-ui-manage.asciidoc (hunk 4 above)
Comment (joepeeples) [reply docs/detections/rules-ui-manage.asciidoc:100]: The toggle label actually never changes to *Disabled* so it's just *Enabled*. But I like the other edit for brevity:
```suggestion
* To enable or disable a single rule, click the rule's *Enabled* toggle.
```

**Review by nastasha-solomon — APPROVED** [review]

File: docs/detections/rules-ui-manage.asciidoc [hunk 5]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
-You can clone, edit, activate, deactivate, and delete rules:
+NOTE: For prebuilt Elastic rules, you can't modify most settings. You can only edit <<rule-schedule, rule actions>> and <<detections-ui-exceptions, add exceptions>>.
 
 . Go to *Detect* -> *Rules*.
 . Do one of the following:
-* Click the *All actions* icon (*...*) on the appropriate row, then select the required action.
-* In the *Rule* column, select all the rules you want to modify, then select the
-required action from the *Bulk actions* menu.
-. To activate or deactivate a rule, click the *Activated* toggle button.
+* Edit a single rule: Click the *All actions* icon (*...*) on a rule, then select *Edit rule settings*. The *Edit rule settings* view opens, where you can modify the <<rules-ui-create, rule's settings>>.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:82]: This is a UX thing; this is technically considered a menu. 
```suggestion
* Edit a single rule: Select the *All actions* menu (*...*) on a rule, then select *Edit rule settings*. The *Edit rule settings* view opens, where you can modify the <<rules-ui-create, rule's settings>>.
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 6]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
-<<rule-schedule, rule actions>>, and <<detections-ui-exceptions, add exceptions>>.
+[float]
+[[manage-rules-ui]]
+=== Manage rules
+
+You can duplicate, enable, disable, and delete rules:
+
+. Go to *Detect* -> *Rules*.
+. Do one of the following:
+* Click the *All actions* icon (*...*) on a rule, then select an action.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:98]:
```suggestion
* Select the *All actions* menu (*...*) on a rule, then select an action.
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 7]
```diff
@@ -2,11 +2,21 @@
 [role="xpack"]
 == Manage detection rules
 
+The Rules page allows you to view and manage all prebuilt and custom detection rules.
+
+[role="screenshot"]
+image::images/all-rules.png[The Rules page]
+
+You can sort the rules by clicking on the *Rule*, *Last updated*, or *Enabled* column header.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:10]:
```suggestion
You can sort the rules by clicking the *Rule*, *Last updated*, or *Enabled* column header. 
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 8]
```diff
@@ -2,11 +2,21 @@
...
 == Manage detection rules
 
+The Rules page allows you to view and manage all prebuilt and custom detection rules.
+
+[role="screenshot"]
+image::images/all-rules.png[The Rules page]
+
+You can sort the rules by clicking on the *Rule*, *Last updated*, or *Enabled* column header. 
+
+TIP: To sort by any other column, enable the *Technical preview* toggle above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:12]: A UX writing thing - we are trying to steer away from "enable/disable the toggle". 

```suggestion
TIP: To sort by any other column, switch the *Technical preview* toggle on above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```

File: docs/detections/rules-ui-manage.asciidoc [hunk 9]
```diff
@@ -60,20 +70,34 @@ image::images/install-prebuilt-settings.png[]
...
+[[manage-rules-ui]]
+=== Manage rules
+
+You can duplicate, enable, disable, and delete rules:
+
+. Go to *Detect* -> *Rules*.
+. Do one of the following:
+* Click the *All actions* icon (*...*) on a rule, then select an action.
+* Select all the rules you want to modify, then select an action from the *Bulk actions* menu.
+* To enable or disable a single rule, click the rule's *Enabled* toggle.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:100]:
```suggestion
* To enable or disable a single rule, switch the rule's *Enabled* toggle on.
```

File: docs/detections/rules-ui-monitor.asciidoc [hunk 10]
```diff
@@ -9,6 +9,10 @@ times, select the *Rule Monitoring* tab on the *Rules* page (*Detect* ->
 [role="screenshot"]
 image::images/monitor-table.png[]
 
+You can sort the rules by clicking on the *Rule* or *Enabled* column header. 
+
+TIP: To sort by any other column, enable the *Technical preview* toggle above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```
Comment (jmikell821) [inline docs/detections/rules-ui-monitor.asciidoc:14]:
```suggestion
TIP: To sort by any other column, switch the *Technical preview* toggle on above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
```

**Review by jmikell821 — COMMENTED** [review]
Just some very minor comments. Looks good, thanks!

**Review by banderror — APPROVED** [review]
LGTM, thank you! 👍

**Pull request merged**